### DISCUSSION TITLE
Is scipy 1.11.2 supported?

### DISCUSSION THREAD

**Discussion by effigies** [networkx/networkx#6868] [Q&A] [answered]
Hi all, I saw that 1.11.0 and 1.11.1 are marked as incompatible:

https://github.com/networkx/networkx/blob/1b28d04e1ea233d506aeb99936e9143b47ffa1b9/pyproject.toml#L68

1.11.2 came out a couple days ago, and I'm curious if I should be avoiding it for networkx compatibility. I haven't been able to follow exactly why the first two were blacklisted, so I can't assess whether the situation has changed or I should stick with 1.10 for now.

**Comment by MridulS** [comment, marked as answer]
There was a bug in scipy 1.11.0 and 1.11.1 affecting us https://github.com/scipy/scipy/issues/18603

It's resolved in 1.11.2 so it should be fine.

**Reply by jarrodmillman** [reply]
Yes, we support 1.11.2 and test against it.

**Reply by jarrodmillman** [reply]
If you are encountering any errors with 1.11.2, please open a bug report. Thanks!!
